src/unwrap.jl

```diff
@@ -235,18 +235,18 @@ function populate_edges!(edges, pixel_image::Array{T, N}, dim, connected, range)
     size_img[dim] -= 1
     idx_step       = fill(0, N)
     idx_step[dim] += 1
-    idx_step_cart  = CartesianIndex{N}(idx_step...)
-    idx_size       = CartesianIndex{N}(size_img...)
+    idx_step_cart  = CartesianIndex{N}(Tuple(idx_step))
+    idx_size       = CartesianIndex{N}(Tuple(size_img))
```

Comment:
This might be better still:
```suggestion
    idx_step_cart  = CartesianIndex{N}(NTuple{N,Int}(idx_step))
    idx_size       = CartesianIndex{N}(NTuple{N,Int}(size_img))
```